utils/httpUtils.ts

```diff
@@ -22,16 +23,35 @@ export const formatUrlParams = (params: { [s: string]: unknown }) => {
 
 export const get = async (url: string, args: any, context: any) => {
   try {
-    const baseURL = process.env.API_URL;
-    const urlPrefix = args.snapshotLinkId ? `/pub/${args.snapshotLinkId}` : "";
-    const response = await fetch(baseURL + urlPrefix + url, {
-      method: "GET",
-      headers: {
-        Cookie: context.request.headers.get("cookie"),
-        "Content-Type": "application/json",
-      },
-    });
-    return await response.json();
+    let baseURL, urlPrefix;
+    const nextGenEnabled = await isNextGenEnabled(context);
+    if (!nextGenEnabled) {
```

Comment:
noticed that the other ones do `nextGenEnabled` first, might be nice to do that here too